dtos/memberships/query-membership.dto.ts

```diff
@@ -15,14 +15,14 @@ export class QueryMembershipDto {
   })
   @IsOptional()
   @IsNumberString()
```

Comment:
```suggestion
```

Reply:
These suggestions made me realize that I could have avoided modifying the dtos.